Automattic/vip-go-mu-plugins
# Skip loading Jetpack for FedRAMP sites by default

## Description

In the `000-vip-init.php` file, when the `VIP_IS_FEDRAMP` constant is true, set the `VIP_JETPACK_SKIP_LOAD` constant to `true` as well.

## Changelog Description

### Jetpack Wrapper: Skip loading Jetpack for FedRAMP sites by default

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Test that Jetpack is enabled for non-FedRAMP sites
    * Browse to `http://$SITE_URL/index.php?rest_route=/jetpack/v4`
        * You should see a list of Jetpack REST API endpoints
    * Browse to `wp-admin` while logged in
        * You should see Jetpack settings in the menu & the options page should function as expected
3. Test that Jetpack is *NOT* enabled for FedRAMP sites
    * Add the following to your `vip-config/vip-config.php` file:
    `define( 'VIP_IS_FEDRAMP', true );`
    * Browse to `http://$SITE_URL/index.php?rest_route=/jetpack/v4`
        * You should see a `404` w/ `rest_no_route` code.
    * Browse to `wp-admin` while logged in
        * You should *NOT* see Jetpack settings in the menu & manually browsing to `http://$SITE_URL/wp-admin/admin.php?page=jetpack` should `403` with a "Sorry, you are not allowed to access this page." message.
